test('esta prueba no debe de fallar',() => {
    if(true){throw new Error('cagadi√±a')}
})